sklearn/externals/_pilutil.py

```diff
@@ -42,13 +42,11 @@
 """
 from __future__ import division, print_function, absolute_import
 
-
 import numpy
-import tempfile
```

Comment:
I don't think we should modify `sklearn/externals`: these are vendored modules, and improvement suggestions should be submitted upstream. Though, in the case of `_pilutil.py`,  I don't think it's possible, since this module is pending removal in scipy.